ase/scripts/generate_ase_environment.py

```diff
@@ -238,12 +240,16 @@ def config_sources(fd, filelist):
             if (sl[-5:] == '.json'):
                 json_srcs.append(s)
 
+    qsys_sim_files = config_qsys_sources(filelist, vlog_srcs)
+
     # List Verilog & SystemVerilog sources in a file
-    if (vlog_found):
+    if (vlog_found or qsys_sim_files):
         fd.write("DUT_VLOG_SRC_LIST = " + VLOG_FILE_LIST + " \n\n")
         with open(VLOG_FILE_LIST, "w") as f:
             for s in vlog_srcs:
                 f.write(s + "\n")
+            if (qsys_sim_files):
+                f.write("-F " + qsys_sim_files + "\n")
 
     # List VHDL sources in a file
     if (vhdl_found):
```

Comment:
What about Qsys flow for VHDL?

Reply:
Qsys is told to generate Verilog here, so the generated files get added to the Verilog file list. VHDL can still instantiate it, right? This is how the old script worked, too.

Reply:
My understanding is --synthesis=VHDL must be passed to generate the right file sets for VHDL. At-least the generated top-level wrapper must be VHDL and .vhd files must be copied over. Unfortunately, we don't have a Qsys VHDL test. Let us revisit it later.